Makefile

```diff
@@ -240,15 +240,15 @@ registry: $(CLIENT_CERT) ## Start a docker registry with certs in the mesos mast
 genconf: start $(CONFIG_FILE) ## Run the DC/OS installer with --genconf.
 	$(RM) dcos-genconf.*.tar ## Remove tar files from previous runs;  otherwise we might skip building Docker image
 	@echo "+ Running genconf"
-	@bash $(DCOS_GENERATE_CONFIG_PATH) --genconf --offline -v
+	@export PORT=${INSTALLER_PORT}; export DCOS_INSTALLER_CONTAINER_NAME=${INSTALLER_CTR}; bash $(DCOS_GENERATE_CONFIG_PATH) --genconf --offline -v
```

Comment:
Each line is a subshell. And shell lets you pass env vars to a command without exporting. So you can make this a bit shorter and line wrap to make this a little more readable:

```
@PORT=${INSTALLER_PORT} \
  DCOS_INSTALLER_CONTAINER_NAME=${INSTALLER_CTR} \
  bash $(DCOS_GENERATE_CONFIG_PATH) --genconf --offline -v
```